examples/flutter_gallery/lib/demo/customized/home.dart

```diff
@@ -0,0 +1,151 @@
+      onWillPop: () {
+        if (_animatedStats) {
+          _scrollController.animateTo(
+            0.0,
+            curve: Curves.easeOut,
+            duration: const Duration(milliseconds: 300),
+          );
+        }
+        return Future<bool>.value(!_animatedStats);
+      },
+      child: Theme(
+        data: _themeData,
+        child: Material(
+          color: const Color(0x00FFFFFF),
+          child: _mainContentWidget,
+        ),
+      ),
+    );
+  }
+
+  void _configureThemes() {
+    _themeData = ThemeData(
+      primaryColor: const Color(0xFF212024),
+      buttonColor: const Color(0xFF3D3D3D),
+      iconTheme: const IconThemeData(color: Color(0xFF4A4A4A)),
+      brightness: Brightness.light,
+    );
+  }
+
+  Widget _contentWidget() {
+    final double screenHeight = MediaQuery.of(context).size.height;
+    final String backTitle = Platform.isAndroid ? 'TrackFit' : '';
+    return Scaffold(
+      backgroundColor: const Color(0xFF212024),
+      body: GlowingOverscrollIndicator(
+        color: const Color(0x00FFFFFF),
+        axisDirection: AxisDirection.down,
+        showLeading: false,
+        showTrailing: false,
+        child: NotificationListener<ScrollNotification>(
+          onNotification: _handleScrollNotification,
+          child: CustomScrollView(
+            controller: _scrollController,
+            physics: SnappingScrollPhysics(midScrollOffset: screenHeight),
+            shrinkWrap: true,
+            slivers: <Widget>[
+              SliverAppBar(
+                pinned: false,
+                title: Text(backTitle),
+                expandedHeight: screenHeight - _kStatsSectionPeekAmount - MediaQuery.of(context).padding.top,
+                leading: const BackButton(
+                  color: Colors.white,
+                ),
+                backgroundColor: const Color(0xFF212024),
+                flexibleSpace: FlexibleSpaceBar(
+                  background: _initialContentWidget,
+                ),
+              ),
+              SliverToBoxAdapter(
+                child: _statsContentWidget,
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  bool _handleScrollNotification(ScrollNotification notification) {
+    final double scrollPosition = notification.metrics.pixels;
+    final double screenHeight = MediaQuery.of(context).size.height -
+        _kStatsSectionPeekAmount - MediaQuery.of(context).padding.top;
+    final double screenOffset = scrollPosition / screenHeight;
+    if (screenOffset >= 0.75 && !_animatedStats) {
+      _animatedStats = true;
+      _initialWelcomeKey.currentState.reverse();
+      _statsContentKey.currentState.animate();
+    } else if (screenOffset < 0.2) {
+      _animatedStats = false;
+      _statsContentKey.currentState.reverse();
+      _initialWelcomeKey.currentState.animate();
+    }
+    return true;
+  }
+
+  /// display the 'stats' content when the 'view my stats' bar is tapped
```

Comment:
Here and everywhere else: make comments full sentences. Capitalize and end with a period.